frontend/lib/ui/footer.tsx

```diff
@@ -1,42 +1,55 @@
 import React from "react";
 import { PrivacyPolicyLink, TermsOfUseLink } from "./privacy-info-modal";
 
-export const Footer: React.FC<{}> = () => (
-  <footer>
-    <div className="container">
-      <div className="columns">
-        <div className="column is-8">
-          <div className="content">
+export const Footer = (props: { pathname?: string }) => {
+  return (
+    <footer>
+      <div className="container">
+        <div className="columns">
+          <div className="column is-8">
+            <div className="content">
+              <p>
+                Disclaimer: The information in JustFix.nyc does not constitute
+                legal advice and must not be used as a substitute for the advice
+                of a lawyer qualified to give advice on legal issues pertaining
+                to housing. We can help direct you to free legal services if
+                necessary.
+              </p>
+              <p>
+                JustFix.nyc is a registered 501(c)(3) nonprofit organization.
+              </p>
+            </div>
+          </div>
+          <div className="column is-4 has-text-right content">
             <p>
-              Disclaimer: The information in JustFix.nyc does not constitute
-              legal advice and must not be used as a substitute for the advice
-              of a lawyer qualified to give advice on legal issues pertaining to
-              housing. We can help direct you to free legal services if
-              necessary.
+              Made with NYC ♥ by the team at{" "}
+              <a href="https://justfix.nyc">JustFix.nyc</a>
             </p>
-            <p>JustFix.nyc is a registered 501(c)(3) nonprofit organization.</p>
+            {/* Include credit for LHI on all ehp routes */}
+            {props.pathname && props.pathname.includes("/ehp") && (
```

Comment:
I wonder, is there a better way to target all routes with the "ehp" prefix?

Reply:
Oh yeah, you should use `props.pathname.startsWith(Routes.locale.ehp[ROUTE_PREFIX])`!  If this kind of thing becomes something we do a lot we can make a new function to make it easier, e.g. `isPathnameInside(pathname, Routes.locale.ehp)`.

Reply:
Hmm so CodeClimate was complaining that this function is now 42 lines and I squelched it, but now that I'm reading the code I wonder if it makes sense to factor out the LHI mention (including the conditional logic) into a separate component?  Eh, up to you.

Reply:
Ahhh yeah... I thought about it but thought that the added complexity of having to pass the pathname prop down wasn't worth it?

Reply:
On the second thought, it would look a bit cleaner. Ok let me add that.